specification/sql/resource-manager/Microsoft.Sql/preview/2015-05-01-preview/blobAuditing.json

```diff
@@ -0,0 +1,299 @@
+{
```

Comment:
Add this file to readme.md in package-pure-2015-05-preview

Reply:
@jaredmoo Done.